types/underscore/underscore-tests.ts

```diff
@@ -474,6 +474,45 @@ _(explicitNumberDictionary).each((value, key, collection) => {
     _.chain(anyValue).forEach(anyCollectionIterator); // // $ExpectType _Chain<any, any>
 }
 
+// get
+
+{
+    // tslint:disable-next-line:no-inferrable-types
+    const num: number = 2;
+
+    // no default value
+    // $ExpectType number
+    _.get({ a: 1 }, 'a');
+
+    // trying to get the property on a simple path
+    // $ExpectType number
+    _.get({ a: 1 }, ['a'], num);
+
+    // trying to get the property on a simple path
+    // $ExpectType number
+    _.get({ a: 1 }, ['a'], num);
+
+    // trying to get the property one level deep
+    // $ExpectType number | { b: number; }
+    _.get({ a: { b: num } }, ['a'], num);
+
+    // trying to get the property one level deep
+    // $ExpectType number | { b: number; }
+    _.get({ a: { b: num } }, ['a', 'b'], num);
+
+    // default value if not found of the 'same type'
+    // $ExpectType number
+    _.get({ a: 1 }, ['b'], num);
+
+    // chained without default value
+    // $ExpectType unknown
+    _({ a: 1 }).get(['b']);
```

Comment:
This is the only case I couldn't get the type "correctly"...

Reply:
Maybe that's due to using separate overloads for with/without fallback value in `UnderscoreStatic` but an optional last parameter in `Underscore`? I suggest trying whether you get a better result (`number`) if you use separate overloads in `Underscore` as well.

Reply:
Thanks. The `U` for the default was getting on the way.